packages/@okta/i18n/src/properties/login.properties

```diff
@@ -819,6 +819,10 @@ oie.verify.google_authenticator.otp.title = Verify with Google Authenticator
 oie.verify.google_authenticator.otp.description = Enter the temporary code generated in your Google Authenticator app
 oie.google_authenticator.otp.enterCodeText = Enter code
 
+## Number Challenge
+oie.numberchallenge.warning = Haven't received a push notification yet? Try opening the Okta Verify app on your device or <$1>resend the push notification</$1>.
```

Comment:
@magizh-okta good call out. The string is slightly different 

```
Haven't received a push notification yet? Try opening the Okta Verify app on your device or <$1>resend the push notification</$1>.
``` 
```
Haven't received a push notification yet? Try opening the Okta Verify App on your phone. 
```
@stevennguyen-okta  Are we ok to use the existing string from V1 ? `Haven't received a push notification yet? Try opening the Okta Verify App on your phone. `. 

The main reason for this is that if a V1 customer has customized this string using the old key `oktaverify.warning` it will be lost if we create a new string for OIE.